contrib/raftexample/raft.go

```diff
@@ -239,9 +239,8 @@ func (rc *raftNode) replayWAL() *wal.WAL {
 	// send nil once lastIndex is published so client knows commit channel is current
 	if len(ents) > 0 {
 		rc.lastIndex = ents[len(ents)-1].Index
-	} else {
-		rc.commitC <- nil
```

Comment:
rc.commitC <- nil  is trigger to loadsnap, at this  is use to trigger loadsnap  at the first time when we start the kvstore. but now we will do loadsnap without the init signal, so i think this is not useful.